imqsauth/http.go

```diff
@@ -304,24 +319,30 @@ func httpSendGroupsJson(w http.ResponseWriter, groups []*authaus.AuthGroup) {
 	httpSendJson(w, jresponse)
 }
 
-func httpSendPermitsJson(central *ImqsCentral, identities []string, ident2perm map[string]*authaus.Permit, w http.ResponseWriter) {
+func httpSendPermitsJson(central *ImqsCentral, users []authaus.AuthUser, ident2perm map[authaus.UserId]*authaus.Permit, w http.ResponseWriter) {
 	//central.Central.Log.Printf("Number of identities %v", len(permits))
 
 	emptyPermit := authaus.Permit{}
 
-	jresponse := make(map[string]*userGroupsResponseJson)
-	for _, identity := range identities {
-		permit := ident2perm[authaus.CanonicalizeIdentity(identity)]
+	jresponse := make(map[string]*userResponseJson)
+	for _, user := range users {
+		permit := ident2perm[user.UserId]
 		if permit == nil {
 			permit = &emptyPermit
 		}
-		jresponse[identity] = &userGroupsResponseJson{}
+		jresponse[user.Email] = &userResponseJson{}
 		groups, err := authaus.DecodePermit(permit.Roles)
 		if err != nil {
 			authaus.HttpSendTxt(w, http.StatusInternalServerError, err.Error())
 			return
 		}
-		jresponse[identity].Groups, err = authaus.GroupIDsToNames(groups, central.Central.GetRoleGroupDB())
+		jresponse[user.Email].Email = user.Email
```

Comment:
Although it will work, I wouldn't get into the habit of performing these kind of table lookups unnecessarily. It's better to write:
```go
jresponse[user.Email] := &userResponseJson{
    Email: user.Email,
    UserId: user.Id,
    // etc
}
```
Each lookup in a map (ie `jresponse[user.Email]`) has a cost, and if there's no good reason to pay that cost, then might as well avoid it.